redis/Redis.php

```diff
@@ -1107,7 +1107,7 @@ public function sAdd( $key, $value1, $value2 = null, $valueN = null ) {}
      *
      * @param   string  $key        Required key
      * @param   array   $values      Required values
-     * @return  boolean The number of elements added to the set
+     * @return  bool The number of elements added to the set
```

Comment:
Isn't it `int`?

Reply:
yep, I also think so ... https://github.com/phpredis/phpredis#sadd

Reply:
The comment is wrong here, see https://github.com/phpredis/phpredis/commit/3491b188e0022f75b938738f7542603c7aae9077#diff-42d0c18a484cc55b889e9d9a2c24d911R1202